Automattic/wp-calypso
# While domain is provisioning, show warning in Google Workspace status

#### Proposed Changes

Follow-up to #64822. When Google Workspace has been purchased for a newly registered domain, users will not see any mailboxes for their subscription (even though they've added one while purchasing the subscription). This is because the back-end process that registers the Google Workspace subscription is waiting for the domain's DNS settings to propagate before the mailbox can be set up.

To avoid confusion for the user, this PR makes it so that the email management screen shows a "warning" status for the Google Workspace subscription if it looks like we are waiting for the DNS settings to propagate for the domain.

![calypso localhost_3000_email_fredrikrombachekelundtest1 live_purchase_fredrikrombachekelundtest1 com(Medium laptop)](https://user-images.githubusercontent.com/1101677/175281205-208236e4-d04f-4a01-b2b5-e4d152efd86d.png)

#### Testing Instructions



1. Register a new domain
2. In the post-checkout upsell, click "Add email"
3. Select Google Workspace on the next screen
4. Purchase a Google Workspace subscription
5. On the post-checkout screen, click "Manage email"
6. Ensure that the next screen displays a "warning" status for the email subscription and a message that says "Configuring domain…"



Related to #
